docs/UserGuide.adoc

```diff
@@ -505,21 +513,21 @@ __Constraints:__
 // tag::list[]
 ==== Listing tasks : `list` +
 
-To go back to the default list of tasks after a round of filtering, use:
+Want to go back to the default list of task after a search? Simply use the list command:
 
 Format: `list`
 // end::list[]
+
 // tag::select[]
 ==== Selecting a task: `select`
 
 To select the task identified by the index number used in the last task listing, use: +
```

Comment:
Do note that the current version of the UG only uses a lead in line followed by the format. There's no statement with "use:". If you think it's necessary and would be too unclear otherwise, let me know, Otherwise, would appreciate if you could scrap this line and replace it with the section in the "note".

Reply:
Rationale: We were told to describe why someone might want to use a command when we introduce it.